report_scripts/consultationByDay_event.sql

```diff
@@ -0,0 +1,54 @@
+DROP PROCEDURE IF EXISTS consultationByDay_event;
+DELIMITER $$
+CREATE PROCEDURE consultationByDay_event(IN org_unit VARCHAR(11))
+BEGIN
+  DECLARE default_group_concat_max_len INTEGER DEFAULT 1024;
+  DECLARE max_group_concat_max_len INTEGER DEFAULT 4294967295;
+  DECLARE date_format VARCHAR(60) DEFAULT '%Y-%m-%d';
+  DECLARE program CHAR(11) DEFAULT 'Rnvvg6utP5O';
+
+SET SESSION group_concat_max_len = max_group_concat_max_len;
```

Comment:
It will be more clear if you could put indentation before every line between 'BEGIN' and 'END'. The same as we do before DECLARE statement.